src/index.ts

```diff
@@ -124,11 +139,14 @@ class TranscriptionEditor {
             Editor();
         }
         if (!window.tinyConfig) {
+            // hide numberded list in line-level editor mode
```

Comment:
tiny typo in comment

```suggestion
            // hide numbered list in line-level editor mode
```